embedded/servo_bq/servo_balsaq.cc

```diff
@@ -50,6 +50,8 @@ void ServoBq::loop() {
   }
 }
 void ServoBq::shutdown() {
+  std::cout << "Servo process shutting down." << std::endl;
+  servos.at(0).shutdown_pwm();
```

Comment:
I'm not sure if this is the best way to do this, because this will disable all servos. But, since our current setup only has 1 balsaq running to manage all 4 servos, it doesn't seem unreasonable.

Reply:
`.at(0)` is all of them, not just the 0th one?

Reply:
the shutdown command applies to the entire driver board, so sending it to any servo shuts down all of them.